### ISSUE TITLE
Forbid to have empty lines before and after the brackets

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#854]
# Rule request

### Wrong

Currently it is possible to write code like this:

```python
some_list = [
    1,
    2,
    3,

]
# Mind ^ the empty line before `]`
```

It shows that no violations are raised for this code (from our linter + dependencies).

We also have this code:

```python
# Mind the empty line after the `[`
some_list = [

    1,
    2,
    3,
]
```

It should not be allowed for consistency reasons. I assume that we should use `tokenize` to check that.

### Correct

But, there are exceptions:

```python
some_list = [
    1,
    2,

    10,
    11,
]
```

Note that a single empty line between elements is allowed (between `2,` and `10,`)

Parens to check: `[`, `(`, `{`, `]`, `)`, `}`

**Comment by lmonyelle** [comment]
I can work on removing some spaces. Is this across all documents or specific ones?

**Comment by sobolevn** [comment]
Hi, @lmonyelle! This is not about deleting empty lines. This is about creating a new linting rule to enforce python code consistency across our users' codebases.

**Comment by QQ-88** [comment]
@sobolevn I'd like to give it a try.

**Comment by sobolevn** [comment]
@QQ-88 you are welcome! Feel free to ask for help.

**Comment by QQ-88** [comment]
Had a look at the code @sobolevn, that's unfortunately above my skill level.

**Comment by sobolevn** [comment]
Ok, no problem!

**Comment by Ben-Drebing** [comment]
I can give this one a try.
